pom.xml

```diff
@@ -769,7 +769,67 @@
 
     </plugins>
     <pluginManagement>
-      <plugins>
+	      <plugins>
```

Comment:
@Salaboy @igdianov I don't know why exactly Eclipse need these extra configurations, but if this solve the problem for Eclipse users, that's ok for me.
@igdianov have you implied the [IDE configuration](https://github.com/Activiti/Activiti/tree/master/ide-configuration/eclipse-configuration) to your IDE? It seems that your PR is not using a different indention.